objectRecognitionSource/src/main/java/io/vantiq/extsrc/objectRecognition/imageRetriever/NetworkStreamRetriever.java

```diff
@@ -47,6 +47,7 @@
     VideoCapture   capture;
     String         camera;
     Logger         log = LoggerFactory.getLogger(this.getClass().getCanonicalName());
+    Boolean        isRTSP = false;
```

Comment:
We might want to call this ```isPushProtocol``` or something more reflective of what it is.  In real life, I think we're differentiating between protocols that push data at us (_e.g._ ```rtsp```) vs. ones from which we have to pull (_e.g._ ```http(s)```).  If we make that differentiation, then it will also help us remember why we did this in the first place :-)